src/ansi/iterator.rs

```diff
@@ -116,19 +119,19 @@ impl<'a> Iterator for AnsiElementIterator<'a> {
     }
 }
 
-// Based on https://github.com/alacritty/vte/blob/0310be12d3007e32be614c5df94653d29fcc1a8b/examples/parselog.rs
+// Based on https://github.com/alacritty/vte/blob/v0.9.0/examples/parselog.rs
 impl vte::Perform for Performer {
-    fn csi_dispatch(&mut self, params: &[i64], intermediates: &[u8], ignore: bool, c: char) {
+    fn csi_dispatch(&mut self, params: &Params, intermediates: &[u8], ignore: bool, c: char) {
         if ignore || intermediates.len() > 1 {
             return;
         }
 
         if let ('m', None) = (c, intermediates.get(0)) {
             if params.is_empty() {
-                // Attr::Reset;
+                // Attr::Reset; TODO: does this need to be handled?
```

Comment:
Technically it can never be empty, but that's not a guarantee I would give. But with the current VTE version this is unreachable. Since even `\e[m` will give you the parameter `[0]`.